pom.xml

```diff
@@ -25,24 +25,12 @@
       <name>Thomas Spengler</name>
     </developer>
   </developers>
-  
-  <distributionManagement>
-    <repository>
-      <uniqueVersion>false</uniqueVersion>
```

Comment:
What do you mean ? 
They are already defined in jenkins parent ? Do I miss something ?